sunpy/net/tests/test_helioviewer.py

```diff
@@ -60,3 +60,29 @@ def test_download_jp2(self, client):
                                        measurement='continuum')
         map_ = sunpy.map.Map(filepath)
         assert isinstance(map_, sunpy.map.GenericMap)
+
+    @skip_glymur
+    def test_download_jp2_permission_error(self, client):
+        """Tests getJP2Image API method"""
+        with pytest.raises(PermissionError):
+            client.download_jp2(
+                '2020/01/01',
+                observatory='SOHO',
+                instrument='MDI',
+                detector='MDI',
+                measurement='continuum',
+                directory='/')
+
+    @skip_glymur
+    def test_download_jp2_directory_not_exist(self, client):
+        """Tests getJP2Image API method"""
+
+        filepath = client.download_jp2(
+            '2020/01/01',
+            observatory='SOHO',
+            instrument='MDI',
+            detector='MDI',
+            measurement='continuum',
+            directory='directorynotexist')
```

Comment:
can you make this inside a tmp dir so it's not related to the path of this code? pytest has a global `tmpdir` fixture, and then this line should be `os.path.join('tmpdir', 'directorynotexist')`